implementations/micrometer-registry-prometheus/src/main/java/io/micrometer/prometheus/PrometheusMeterRegistry.java

```diff
@@ -295,8 +296,14 @@ public DistributionSummary newDistributionSummary(Meter.Id id,
 
                 }
 
-                samples.add(
-                        new Collector.MetricFamilySamples.Sample(conventionName + "_count", tagKeys, tagValues, count));
+                if (lastExemplar == null) {
+                    samples.add(new Collector.MetricFamilySamples.Sample(conventionName + "_count", tagKeys, tagValues,
+                            count));
+                }
+                else {
+                    samples.add(new Collector.MetricFamilySamples.Sample(conventionName + "_count", tagKeys, tagValues,
+                            count, copyExemplarWithNewValue(1.0, lastExemplar)));
```

Comment:
Here I'm setting a new value for the `Exemplar`. This is needed because the value of the last recorded Exemplar is the last recorded (and sampled) value by `PrometheusDistributionSummary`/`PrometheusTimer`. That conforms the "contract" for being the value of the `.lastExemplar()` (which can be used elsewhere, i.e. in `_sum`) but for `_count` the value should be `1.0`.